.github/workflows/release.yaml

```diff
@@ -81,3 +81,83 @@ jobs:
           arch: amd64
           wheels-key: ${{ secrets.WHEELS_KEY }}
           requirements: "requirements.txt"
+
+  release-landing-page:
+    name: Release landing-page frontend
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v4.2.2
+      - name: Setup Node
+        uses: actions/setup-node@v4.1.0
+        with:
+          node-version-file: ".nvmrc"
+          cache: yarn
+      - name: Install dependencies
+        run: yarn install
+      - name: Download Translations
+        run: ./script/translations_download
+        env:
+          LOKALISE_TOKEN: ${{ secrets.LOKALISE_TOKEN }}
+      - name: Build landing-page
+        run: landing-page/script/build_landing_page
+      - name: Checkout landingpage repo
+        uses: actions/checkout@v4.2.2
+        with:
+          repository: home-assistant/landingpage
+          path: landingpage-repo
```

Comment:
IMHO this should be done the other way around.

Landingpage should be responsible for itself and care about its own; this logic should thus live in the landing page repo, not frontend repo.